addons/mail/static/src/components/discuss_sidebar_mailbox/discuss_sidebar_mailbox.xml

```diff
@@ -1,23 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
     <t t-name="mail.DiscussSidebarMailbox" owl="1">
-        <div class="o_DiscussSidebarMailbox"
+        <button class="o_DiscussSidebarMailbox btn d-flex align-items-center py-1 px-0 border-0 rounded-0 font-weight-normal text-reset"
```

Comment:
`text-reset` is quite problematic, because it makes part of styling of this component coupled with a context outside of this component.

The better approach would be to clearly specific why are the expected coloring of the text.

Reply:
Is the `text-reset` introduced because there is an undesirable change of text colour with `<button>` and `btn`?

If that's the case, would be better to explicitly set the right text colour here.

Reply:
Indeed, `text-reset` was introduced to make the element inherit the colour of the parent, and avoiding colour changes by using `<button>` + `.btn`

> Is the `text-reset` introduced because there is an undesirable change of text colour with `<button>` and `btn`?
> 
> If that's the case, would be better to explicitly set the right text colour here.

Indeed, `text-reset` was introduced to make the element inherit the colour of the parent, and avoiding colour changes by using `<button>` + `.btn`.

In this case, I'll change the class to `text-dark`. 👍

Reply:
Better, thanks :)